coderd/cryptokeys/dbkeycache.go

```diff
@@ -61,18 +76,39 @@ func NewDBCache(logger slog.Logger, db database.Store, feature database.CryptoKe
 	return d
 }
 
-// Verifying returns the CryptoKey with the given sequence number, provided that
+func (d *dbCache) EncryptingKey(ctx context.Context) (id string, key interface{}, err error) {
+	return d.latest(ctx)
+}
+
+func (d *dbCache) DecryptingKey(ctx context.Context, id string) (key interface{}, err error) {
+	return d.sequence(ctx, id)
+}
+
+func (d *dbCache) SigningKey(ctx context.Context) (id string, key interface{}, err error) {
+	return d.latest(ctx)
+}
+
+func (d *dbCache) VerifyingKey(ctx context.Context, id string) (key interface{}, err error) {
+	return d.sequence(ctx, id)
+}
```

Comment:
I know `jwtutils` is the user of this package, and handles the `interface{}`, but it is unfortunate to need to use `interface{}` :cry:.

I took a stab at throwing in generics to get type safety, but it locks in a feature to a specific algo.

Reply:
Yeah it is :(